python/feedwatchlist_other.py

```diff
@@ -0,0 +1,29 @@
+#This file is partly auto-generated
+
+#!/usr/bin/python3
+
+"""
+    feedwatchlist_other.py
+
+    MediaWiki Action API Code Samples
+    Demo of `Feedwatchlist` module: Get a watchlist feed from another user.
+
+    MIT License
+"""
+
+import requests
+
+S = requests.Session()
+
+URL = "https://en.wikipedia.org/w/api.php"
+
+PARAMS = {
+    "action": "feedwatchlist",
+    "wlowner": "sample_user",
+    "wltoken": "sample_watchlist_token"
+}
+
+R = S.get(url=URL, params=PARAMS)
+DATA = R.json()
```

Comment:
I tested this code, and it looks like we won't be able to generate other user's watchlist with our watchlist token. This is not very clear and super confusing in the original docs, and we should address this in the doc in progress what are the alternatives to this. The user experience around obtaining a watchlist token on MediaWiki.org (https://www.mediawiki.org/wiki/Special:Preferences#mw-prefsection-watchlist) is confusing too; I think we should create a task about this on Phabricator.

One TODO here would also be to change `R.json()`
 to support fetching the output which will be in XML format and not json, unlike the other API modules.

Reply:
Woops! I swear an earlier version had R.text there. Updated in renamed file, get_other_watchlist_feed.py